packages/editor/src/store/private-selectors.js

```diff
@@ -75,6 +75,9 @@ export const getInsertionPoint = createRegistrySelector( ( select ) =>
 export function getListViewToggleRef( state ) {
 	return state.listViewToggleRef;
 }
+export function getInserterSidebarToggleRef( state ) {
+	return state.inserterSidebarToggleRef;
```

Comment:
I know this is an existing pattern, but I just want to share my concerns about this approach again. I still think storing non-serializable data in the store is a [bad practice](https://redux.js.org/style-guide/#do-not-put-non-serializable-values-in-state-or-actions).

First of all, it breaks the Redux Devtools; I think I have had to disable the extension ever since we merged this approach the first time. Even if we special case this by making it serializable in the [extension settings](https://github.com/reduxjs/redux-devtools/blob/main/extension/docs/API/Arguments.md#serialize), we can't make sure that new features of the extensions will continue support this pattern.

Second, I think using a context provider is a much simpler and intuitive solution. It might mean an extra wrapper and private settings in the `editor` component, but it also matches more closely to what React expects. We can also simply group the same patterns over and over again and create an abstraction around it, so it feels less "hacky."

This is not a blocker and I just want to re-share this for visibility 😆.